datahub-web-react/src/app/identity/group/AssignRoletoGroupConfirmation.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { message, Popconfirm } from 'antd';
+import { useBatchAssignRoleMutation } from '../../../graphql/mutations.generated';
+import { DataHubRole } from '../../../types.generated';
+import analytics, { EventType } from '../../analytics';
+
+type Props = {
+    visible: boolean;
+    roleToAssign: DataHubRole | undefined;
+    groupName: string;
+    groupUrn: string;
+    onClose: () => void;
+    onConfirm: () => void;
+};
+
+export default function AssignRoletoGroupConfirmation({
+    visible,
+    roleToAssign,
+    groupName,
+    groupUrn,
+    onClose,
+    onConfirm,
+}: Props) {
+    const [batchAssignRoleMutation] = useBatchAssignRoleMutation();
+    // eslint-disable-next-line
+    const batchAssignRole = () => {
+        batchAssignRoleMutation({
+            variables: {
+                input: {
+                    roleUrn: roleToAssign?.urn,
+                    actors: [groupUrn],
+                },
+            },
+        })
+            .then(({ errors }) => {
+                if (!errors) {
+                    analytics.event({
+                        type: EventType.SelectUserRoleEvent,
```

Comment:
Please add new event name for group. There is 1 place in java where the new event is required to be added DataHubUsageEventType. It is a 1 line change. You can talk with @pedro93 and he can make that change on the java side.